api/lib/infrastructure/repositories/sco-certification-candidate-repository.js

```diff
@@ -47,3 +39,13 @@ function _scoCandidateToDTOForSession(sessionId) {
     };
   };
 }
+
+async function _batchInsertCandidatesAndCoreSubscriptions(candidatesToBeEnrolledDTOs) {
+  await knex
+    .batchInsert('certification-candidates', candidatesToBeEnrolledDTOs)
+    // eslint-disable-next-line knex/avoid-injections
+    .returning([knex.raw('id as "certificationCandidateId"'), knex.raw(`'${SubscriptionTypes.CORE}' as type`)])
```

Comment:
Le disable de la règle Eslint permet d'utiliser notre enum `SubscriptionTypes`.

Si on préfère éviter d'enfreindre cette règle, préférer écrire la valeur en dure : 
```suggestion
.returning([knex.raw('id as "certificationCandidateId"'), knex.raw("'CORE' as type")])
```